sympy/printing/pretty/pretty.py

```diff
@@ -531,7 +531,7 @@ def adjust(s, wid=None, how='<^>'):
         for lim in expr.limits:
             if len(lim) == 3:
                 prettyUpper = self._print(lim[2])
-                prettyLower = self._print(Equality(lim[0], lim[1]))
+                prettyLower = self._print(str(lim[0]) + " = " + str(lim[1]))
```

Comment:
@valglad , S.NegativeInfinity , S.Infinity, S.One and S.Zero doesnot have .name property . So, maybe we have to find any other solution